doc/Language/classtut.pod6

```diff
@@ -633,6 +521,41 @@ making dinner
 eating dinner. NOM!
 =end code
 
+=head2 A word on types
```

Comment:
OK; I see it's been moved here. It probably makes sense. it creates a new header, though...